npm/src/index.ts

```diff
@@ -50,26 +50,18 @@ export const renderModules = () => {
   if (uploadContainer) {
     uploadContainer.removeAttribute("hidden")
     const frontEndInfo = getFrontEndInfo()
-
     getKeycloakInstance().then((keycloak) => {
-      fetch(`${frontEndInfo.uploadUrl}/cookies`, {
```

Comment:
Does this fetch still need to be run in a loop somehow, so that the upload can continue once the initial cookies have expired?

Reply:
It is run in a loop, it's in here https://github.com/nationalarchives/tdr-transfer-frontend/blob/master/npm/src/auth/index.ts#L48

Reply:
Ah, I got confused because `scheduleTokenRefresh` sounded like it just refreshed the Keycloak token. Maybe it needs a name that makes it clear it refreshes all the credentials?